test/symlink.js

```diff
@@ -183,44 +184,54 @@ describe('symlink stream', function() {
     stream.end();
   });
 
-  it('should write buffer files to the right folder', function(done) {
-    var inputPath = path.join(__dirname, './fixtures/test.coffee');
-    var inputBase = path.join(__dirname, './fixtures/');
-    var expectedPath = path.join(__dirname, './out-fixtures/test.coffee');
-    var expectedContents = fs.readFileSync(inputPath);
-    var expectedBase = path.join(__dirname, './out-fixtures');
-    var expectedMode = parseInt('677', 8)  & ~process.umask();
-
-    var expectedFile = new File({
-      base: inputBase,
-      cwd: __dirname,
-      path: inputPath,
-      contents: expectedContents,
-      stat: {
-        mode: expectedMode,
-      },
-    });
-
-    var buffered = [];
+  [{
```

Comment:
I prefer not to have generative testing.  It allows for separate refactoring of tests